src/lib/rest/ConnectionInfo.cpp

```diff
@@ -211,7 +210,28 @@ ConnectionInfo::~ConnectionInfo()
   servicePathV.clear();
 }
 
-void ConnectionInfo::modify(const std::string &_url, const std::string &_verb, const std::string &_payload, const HttpHeaders &head)
+void ConnectionInfo::reset()
+{
+  version = "HTTP/1.1";
+  servicePath = "/";
+
+  httpHeaders.gotHeaders = true;
+  httpHeaders.userAgent = "orionWS/0.1";
+  httpHeaders.accept = "*/*";
+  httpHeaders.contentLength = 0;
+  httpHeaders.servicePath = "/";
+  httpHeaders.tenant.clear();
+
+  uriParam["details"] = "off";
```

Comment:
I think we have constant for "details", "limit", etc. in `#define` in some .h file. Not sure, but it is worth to have a look and, in that case, use them.